src/main/scala/ai/privado/languageEngine/java/language/package.scala

```diff
@@ -36,12 +36,12 @@ package object language {
 
   implicit class StepsForProperty(val trav: Traversal[JavaProperty]) extends AnyVal {
 
-    def usedAt: Traversal[CfgNode] = trav.out(EdgeTypes.IS_USED_AT).cast[CfgNode]
+    def usedAt: Traversal[AstNode] = trav.out(EdgeTypes.IS_USED_AT).cast[AstNode]
```

Comment:
Do we really need to make this change `CfgNode` => `AstNode` and why?

Reply:
@khemrajrathore can answer this better.